x-pack/plugins/security_solution/public/overview/components/detection_response/recent_cases_table/status_badge.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { CaseStatuses } from '@kbn/cases-plugin/common';
+import { EuiBadge } from '@elastic/eui';
+
+import * as i18n from '../translations';
+
+interface Props {
+  status: CaseStatuses;
+}
+
+export const StatusBadge: React.FC<Props> = ({ status }) => {
+  return (
+    <EuiBadge color={statuses[status].color} data-test-subj="case-status-badge">
+      {statuses[status].label}
+    </EuiBadge>
+  );
+};
+
+StatusBadge.displayName = 'StatusBadge';
+
+const statuses = {
```

Comment:
This is a constant, it should be moved to the top and it would also be good to add the `as const` at the end to make sure it is read-only and never modified.

Reply:
fixed!